app/policies/kudo_policy.rb

```diff
@@ -0,0 +1,34 @@
+class KudoPolicy < ApplicationPolicy
+  def initialize(employee, kudo)
```

Comment:
You are inheriting this method from `ApplicationPolicy`, there is no need to re-write it here

Reply:
I had to change back variable names employee and kudo to user and record in ApplicationPolicy (now I can use it for other than kudo policy), so I didn't remove initialize method from KudoPolicy.